pkg/network/apis/network/types.go

```diff
@@ -1,12 +1,15 @@
 package network
 
 import (
+	"time"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	ClusterNetworkDefault       = "default"
 	EgressNetworkPolicyMaxRules = 50
+	DefaultInformerResyncPeriod = 30 * time.Minute
```

Comment:
pkg/network/apis/network/ is public; it's for things that external software might need to use. Stuff that is only used by other parts of origin goes in pkg/network/plugin.go. (They used to be all mixed up together but this got fixed recently along with the other reorgs.)

(OTOH it's not clear to me that we should be using our own default value here anyway, rather than picking up some shared origin-wide default from somewhere else...)

Reply:
OpenShift is using 10 mins as informer resync period and k8s is using 15 mins. These values seems aggressive to me. Resync is intended to help the case where you failed to process the first time due to some transient error and hoping that processing again after sometime will succeed. If the watch code is smart enough to recognize nothing has changed, frequent resync is okay but if we are unnecessarily updating iptables, etc. then I think we should resync less often. Moved 30 min resync period constant to pkg/network/plugin.go